cms/djangoapps/models/settings/course_metadata.py

```diff
@@ -64,6 +64,10 @@ def filtered_list(cls):
         if not settings.FEATURES.get('ENABLE_VIDEO_UPLOAD_PIPELINE'):
             filtered_list.append('video_upload_pipeline')
 
+        # Do not show facebook_url if the feature is disabled.
```

Comment:
Just a general question. Why do we use "if not True/False" instead of "if True/False"?